core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournalConfig.scala

```diff
@@ -45,12 +45,35 @@ class CassandraJournalConfig(system: ActorSystem, config: Config) extends Cassan
     }(collection.breakOut)
   }
 
+  private val tags: HashMap[String, Int] = loadTagMap("tags")
+
+  private val useTagPrefixes: Boolean = config.getBoolean("use-tag-prefixes")
+
+  private val tagPrefixes: HashMap[String, Int] = loadTagMap("tag-prefixes")
```

Comment:
I'd prefer one single config section with support for suffix wildcard.
```
tags {
  Blog* = 1
  Announcement = 2
  Authors = 1
}
```

when reading this config we can look if the entry key ends with "*" and then put it into `tagPrefixes`, otherwise ordinary `tags`.

And the use-tag-prefixes shouldn't be needed, `tagPrefixes` Map will be empty if no such configured.

Reply:
I like the idea. My only concern is backward compatibility in case someone used tags with `*` at the end. That's why I've split tagging algorithms using a flag. Otherwise I really like what you've proposed.

Reply:
yeah, I think that is a small risk that we can live with

Reply:
Ok